File: Modul 6/C/PRAK601-2310817220025-FirdaKhoirunisa.c
#include<stdio.h>

int main() {
    int baris, kolom, i, j;
    scanf("%d %d", &baris, &kolom);
    int matrix[baris][kolom];

    for (i = 0; i < baris; i++) {
        for (j = 0; j < kolom; j++) {
            scanf("%d", &matrix[i][j]);
        }
    }
    for (i = 0; i < baris; i++) {
        for (j = 0; j < kolom; j++) {
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul 6/Python/PRAK601-2310817220025-FirdaKhoirunisa.py
baris, kolom = map(int, input().split())
k = list(map(int, input().split()))
matriks = []
for i in range(baris):
    row = []
    for j in range(kolom):
       row.append(k[i * kolom + j])
    matriks.append(row)
print(" ")
for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end = " ")
    print()